core-charter.txt

```diff
@@ -1,40 +1,65 @@
+The CoRE Working Group (WG) provides a framework for resource-oriented applications intended to run on constrained IP networks. A constrained IP network may have additional constraints compared to normal IP networks such as limited packet sizes, and a high degree of packet loss. These networks and the nodes within them are characterized by severe limits on throughput, available power, and particularly on the complexity that can be supported with limited code size or limited RAM per node [RFC 7228]. Constrained networks can occur as part of home and building automation, energy management, and the Internet of Things.
 
 The CoRE WG will define a framework for a limited class of applications: those that deal with the manipulation of simple resources on constrained networks. This includes applications to monitor simple sensors (e.g., temperature sensors, light sensors, and power meters), to control actuators (e.g., light switches, heating controllers, and door locks), and to manage devices.
 
-The general architecture targeted by the CoRE WG framework consists of nodes on the constrained network, called Devices, that are responsible for one or more Resources that may represent sensors, actuators, combinations of values, and/or other information. Devices send messages to change and query resources on other Devices. A Device can send notifications about changed resource values to Devices that have expressed their interest to receive notification about changes. A Device can also publish or be queried about its resources. Typically, a single physical host on the network would have just one Device but a host might represent multiple logical Devices. The specific terminology to be used here is to be decided by the working group.
+The general architecture targeted by the CoRE WG framework consists of nodes on the constrained network, called Devices, that are responsible for one or more Resources that may represent sensors, actuators, combinations of values, and/or other information. Devices send messages to change and query resources on other Devices. A Device can send notifications about changed resource values to Devices that have expressed their interest to receive notification about changes. A Device can also publish or be queried about its resources. 
```

Comment:
Can't we just refer to RFC7228 here?  Do we need to explain this again and again?

Reply:
7228 would be a great reference for "nodes on the constrained network", but constrainedness is not what this paragraph is about.